stable/mediawiki/templates/ingress.yaml

```diff
@@ -3,10 +3,10 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | trimSuffix "-" -}}"
+  name: {{ template "mediawiki.fullname" $ }}
```

Comment:
why the `$` sign? Shouldn't that be a `.` ?

Reply:
It's because of the range